lib/modules/hosts/streamvi.js

```diff
@@ -5,9 +5,12 @@ import { Host } from '../../core/host';
 export default new Host('streamvi', {
 	name: 'streamvi',
 	domains: ['streamvi.com'],
-	logo: 'https://streamvi.com/favicon.ico',
-	detect: ({ pathname }) => (/^\/([^\/]+)$/i).exec(pathname),
-	handleLink(href, [, code]) {
+	logo: 'https://streamvi.com/assets/logo.png',
+	detect: ({ searchParams }) => {
+		const code = searchParams.get('video') || '';
+		return [code.toString()];
```

Comment:
If no code is found, this should return false-ish to prevent the expando being build.